File: C/Strings/1049 - Animal..c
/*
* Autor: Pierre Vieira
* Data de submissão: 03/10/2018 19:45:44
*/
#include <stdio.h>
int main(){
    char string1[30], string2[30], string3[30];
    scanf(" %[^\n]s", string1);
    if (string1[0] == 'v'){
       scanf(" %[^\n]s", string2);
       if(string2[0] == 'a'){
         scanf(" %[^\n]s", string3);
         if(string3[0] == 'c'){
            printf("aguia\n");
            }
          else if(string3[0] == 'o'){
                  printf("pomba\n");
         }
       }
       else if(string2[0] == 'm'){
            scanf(" %[^\n]s", string3);
            if(string3[0] == 'o'){
                printf("homem\n");
            }
            else if(string3[0] == 'h'){
                printf("vaca\n");
            }
       }
    }
    else if(string1[0] == 'i'){
        scanf(" %[^\n]s", string2);
        if(string2[0] == 'i'){
            scanf(" %[^\n]s", string3);
            if(string3[2] == 'm'){
                printf("pulga\n");
            }
            else if(string3[2] == 'r'){
                printf("lagarta\n");
            }
        }
        else if(string2[0] == 'a'){
            scanf(" %[^\n]s", string3);
            if(string3[0] == 'h'){
                printf("sanguessuga\n");
            }
            else if(string3[0] == 'o'){
                printf("minhoca\n");
            }
        }
    }
}

File: C#/Iniciante/1049 - Animal.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 16/05/2020 13:17:15
*/
using System;
using System.Globalization;
using Microsoft.VisualBasic;

namespace URI {
    class URI {
        static void Main(string[] args) {
            string vertebrado, tipo, come;
            vertebrado = Console.ReadLine();
            tipo = Console.ReadLine();
            come = Console.ReadLine();
            if (vertebrado == "vertebrado") {
                if (tipo == "ave") {
                    if (come == "carnivoro") {
                        Console.WriteLine("aguia");
                    }
                    else {
                        Console.WriteLine("pomba");
                    }
                }
                else {
                    if (come == "onivoro") {
                        Console.WriteLine("homem");
                    }
                    else {
                        Console.WriteLine("vaca");
                    }
                }
            }
            else {
                if (tipo == "inseto") {
                    if (come == "hematofago") {
                        Console.WriteLine("pulga");
                    }
                    else {
                        Console.WriteLine("lagarta");
                    }
                }
                else {
                    if (come == "hematofago") {
                        Console.WriteLine("sanguessuga");
                    }
                    else {
                        Console.WriteLine("minhoca");
                    }
                }
            }
        }
    }
}
